### PR TITLE
BACK-4796: Prefix testnet token ticker with "t" in transaction config…

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

See https://ledgerhq.atlassian.net/wiki/spaces/~623066f962dc1e006801f92c/pages/4071784452/BACK-4796+Ledger+Live+signature+mismatch+for+Go+rli+tokens

https://github.com/LedgerHQ/ledger-live/pull/2690

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by jnicoulaud-ledger** [comment]
@henrily-ledger: Updated signatures in #2907

File: libs/ledgerjs/script/crypto-assets-importer/importers/erc20-signatures.js [hunk 1]
```diff
@@ -38,14 +34,19 @@ module.exports = {
       readFileJSON(path.join(folder, id, "common.json")),
       readFileJSON(path.join(signatureFolder, id, "ledger_signature.json")),
     ]).then(([common, ledgerSignature]) => {
+
+      const currency = getCryptoCurrencyById(path.basename(path.dirname(folder)));
+
+      // match crypto-assets convention for tickers: testnet tokens are prefixed with "t"
+      // https://github.com/LedgerHQ/crypto-assets/blob/d2fe1cf9a110614650191555b846a2e43eb67b8f/scripts/hsm/coin_parameters/coin_parameters.py#L163
+      const prefix = currency.isTestnetFor !== undefined ? 't': '';
```
[the remark this answers is not in the document]
Comment (henrily-ledger) [reply libs/ledgerjs/script/crypto-assets-importer/importers/erc20-signatures.js:42]: No is not I guess

**Comment by adrienlacombe-ledger** [comment]
IT seems that https://github.com/LedgerHQ/ledger-live/pull/2828/files#diff-ddf7ce79297278369f2dced95df9c3ee3ef5257283480762df58bf1a122bdcafR52 updates tickers, which was never done before. Also, this test https://github.com/LedgerHQ/ledger-live/actions/runs/4446143749/jobs/7806099220#step:9:4776 precisely check for ticker consistently therefore failing as expected. I see 2 options here: update of the test suite to remove this check for testnet tokens OR update the tickers on both sides i.e. in the CAL as well.

**Review by gre — CHANGES_REQUESTED** [review]
there is a new file ethereum.json that is  381,179 line of code, i'm not sure we want it in?

**Comment by jnicoulaud-ledger** [comment]
@gre yes, that is a mistake, I will update this PR later today

**Comment by jnicoulaud-ledger** [comment]
This should be better:

- rebased
- removed extra `ethereum.json`
- fixed test

Note: I had to run importers using the latest CAL (https://github.com/LedgerHQ/crypto-assets/commit/e97b76d6d9fa217ad70dc08a03f4a97ea00008ea), so this includes a CAL update as well, this is why there more are changes than just testnet signatures.

**Comment by jnicoulaud-ledger** [comment]
Test now failing because firmware upgrade prompt is not shown on linux:
image
I don't think it's related to this PR

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**